services/newsfeed/config_drupal.js

```diff
@@ -15,51 +15,30 @@
  */
 const getDrupalConfig = ({ enabled, name, url, filter }) => ({
 	name,
-	type: 'rest',
 	enabled,
 	url,
-	interval: 5 * 60, // seconds
+	table: 'newsfeed',
 	newsfeed: {
 		mapper: {
 			source: '=,string',
 			source_id: 'nid,string',
 			hash: '=,string',
 			title: '=,string',
-			content: 'description,string',
+			content_orig: 'description,string',
+			content_t: '=,string',
 			url: 'link,string',
 			image_url: '=,string',
-			timestamp: '=',
+			ctime: '=,number',
+			mtime: 'ctime,number',
```

Comment:
`mtime` cannot always be the same as `ctime`. `ctime` is the post creation time and `mtime` is the last modified time. They can be equal initially though.